cnd/src/config/mod.rs

```diff
@@ -91,6 +94,28 @@ pub struct Parity {
     pub node_url: reqwest::Url,
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct Lnd {
+    pub http_rpc_socket: Socket,
+    pub macaroon: Option<PathBuf>,
+}
+
+impl Default for Lnd {
+    fn default() -> Self {
+        Self {
+            http_rpc_socket: default_lnd_http_rpc_socket(),
+            macaroon: None,
+        }
+    }
+}
+
+fn default_lnd_http_rpc_socket() -> Socket {
+    Socket {
+        address: IpAddr::V4(Ipv4Addr::UNSPECIFIED),
+        port: 443,
```

Comment:
```suggestion
        port: 8080,
```
I think the default port is actually 8080.

Reply:
TLS is on by default so I assumed the default was 443.  The default config suggests both.  I also thought using 443 makes it explicit that LND use TLS (perhaps with this in mind I should have noticed the TLS cert was missing :)  The LND REST API docs use 8080 as the example port so I will do the same.

Reply:
TLS == 443 only applies to web servers I believe :)

Reply:
> TLS == 443 only applies to web servers I believe :)

Cool, I did not know that.